src/shims/windows/foreign_items.rs

```diff
@@ -8,6 +8,7 @@ use rustc_target::spec::abi::Abi;
 use crate::*;
 use shims::foreign_items::EmulateByNameResult;
 use shims::windows::sync::EvalContextExt as _;
+use smallvec::SmallVec;
```

Comment:
This is not at all perf critical, no need to introduce a new dependency.

Reply:
There already is a dependency on `smallvec` so I just used it. The alternative would be to use `Vec` for the `collect` call in 149-157, but that also requires a new `use`.

Really, I'd just want to `collect` into a `[Size; 11]`, but you can't do that 😞

Reply:
Oh, sorry, I saw the Cargo.toml edit and then thought this was a new dependency. In that case :shrug: 

> I'd just want to collect into a [Size; 11], but you can't do that 

Yes! I was thinking the same. ;) Is there a tracking issue for that? Something something const generics should make it possible...